pkgs/servers/sql/postgresql/default.nix

```diff
@@ -1,10 +1,11 @@
-{ lib, stdenv, glibc, fetchurl, zlib, readline, libossp_uuid, openssl, libxml2, makeWrapper, tzdata }:
+{ lib, stdenv, glibc, fetchurl, zlib, readline, libossp_uuid, openssl, libxml2, makeWrapper, tzdata, systemd }:
 
 let
 
   common = { version, sha256, psqlSchema }:
    let atLeast = lib.versionAtLeast version; in stdenv.mkDerivation (rec {
     name = "postgresql-${version}";
+    inherit version;
```

Comment:
Doesn't this cause a mass rebuild on all downstream-dependent `postgresql` packages (e.g. libpq clients)? I was pretty sure changing this on my own build causes a full rebuild since the 'version' attribute exists, though I can't remember exactly *why*.

Reply:
Good point. I didn't bother, but for 18.09 we probably should. I will cherry-pick your commit to 18.09 as it's basically the same as mine, minus the mass-rebuild.

Reply:
Cool, thanks!